src/app/Scenes/MyCollection/Components/MyCollectionArtistsPrompt/MyCollectionArtistsAutosuggestItem.tsx

```diff
@@ -0,0 +1,79 @@
+import { Avatar, Button, CheckIcon, Flex, Text, useSpace } from "@artsy/palette-mobile"
+import { MyCollectionArtistsAutosuggestItem_artist$key } from "__generated__/MyCollectionArtistsAutosuggestItem_artist.graphql"
+import { MyCollectionAddCollectedArtistsStore } from "app/Scenes/MyCollection/Screens/MyCollectionAddCollectedArtists/MyCollectionAddCollectedArtistsStore"
+import { MotiView } from "moti"
+import { FC } from "react"
+import { graphql, useFragment } from "react-relay"
+
+interface MyCollectionArtistsAutosuggestItemProps {
+  artist: MyCollectionArtistsAutosuggestItem_artist$key
+}
+
+export const MyCollectionArtistsAutosuggestItem: FC<MyCollectionArtistsAutosuggestItemProps> = ({
+  artist,
+}) => {
+  const space = useSpace()
+  const data = useFragment(QUERY, artist)
```

Comment:
Suggestion to be able to call the artist from the fragment `artist`. This might be easier to read than the more generic `data`.

```suggestion
export const MyCollectionArtistsAutosuggestItem: FC<MyCollectionArtistsAutosuggestItemProps> = (props) => {
  const space = useSpace()
  const artist = useFragment(QUERY, props.artist)
```

Reply:
I think data is an okay pattern, we have it like this in our workshop docs, all apps use the same pattern calling the `key` the type name and it is like Relay uses in their docs, example https://relay.dev/docs/api-reference/use-fragment/#usefragment

Reply:
The pattern is ok and working, but in my opinion, it has two significant drawbacks:

1. It creates unnecessary ambiguity when accessing the data further down in the component, making it more difficult to understand. (e.g. `data.internalID` vs `artist.internalID`).
2. Refactoring/renaming is needed when adding more fragment to the component ([example](https://github.com/artsy/eigen/blob/12b44d2e419b55e1ae95466df7b6eac870e1a056/src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/RequestForPriceEstimate/RequestForPriceEstimateBanner.tsx#L23)).

Reply:
In Eigen's codebase, there can be found a great mix of both patterns: 59 x `data = useFragment`, 54 x `somethingData = useFragment`` and 65 x `something = useFragment`.